src/Common/src/System/Net/IPv6AddressHelper.Common.cs

```diff
@@ -303,11 +298,14 @@ internal static unsafe void Parse(ReadOnlySpan<char> address, ushort* numbers, i
                             numberIsValid = false;
                         }
 
-                        start = i;
+                        start = i + 1;
```

Comment:
Why does this change?

Reply:
Before this change this internal `Parse` method used to return `ref string scopeId` with redundant leading '%' symbol so this +1 basically removes it.
Then it was ignored here https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/src/System/Net/IPAddressParser.cs#L222 as you can see `for` starts from `1`.
if I let this method to return `%` then I'll have to trim it here (with allocations) https://github.com/dotnet/corefx/pull/35278/files#diff-457efef691232b0d4ae39c67298be093R221